src/ng-select/ng-select.component.ts

```diff
@@ -592,7 +592,12 @@ export class NgSelectComponent implements OnDestroy, OnChanges, AfterViewInit, C
 
     private _handleTab(_: KeyboardEvent) {
         if (this.isOpen) {
-            this.close();
+            if (this.itemsList.markedItem) {
```

Comment:
Lets make it configurable, you can add new input `tabSelectsValue` which by default can be true so tab will select value by default.

Reply:
I would suggest `selectOnTab`

Reply:
@varnastadeus i agree